index.js

```diff
@@ -2125,6 +2125,26 @@
     return Extend.methods.extend(extend_)(f);
   }
 
+  //# duplicate :: Extend w => w a -> w (w a)
+  //.
+  //. Adds one level of nesting to a comonadic structure.
+  //.
+  //. This function is derived from [`extend`](#extend).
+  //.
+  //. ```javascript
+  //. > duplicate(Identity(1))
+  //. Identity(Identity(1))
+  //.
+  //. > duplicate([[1, 2, 3]])
+  //. [[[1, 2, 3]]]
+  //.
+  //. > duplicate([1, 2, 3])
+  //. [[1, 2, 3], [2, 3], [3]]
+  //. ```
```

Comment:
I think the order of the second and third examples should be switched, so the unnested example is presented before the nested one. Do you agree?

Reply:
That was intentional as I thought the third example might be non-intuitive. I'm not married to that order though.

Reply:
I found the third example surprising, but perhaps more so due to first having read the second example, which hid the surprising behaviour from me. Emphasizing the behaviour for singleton arrays felt disingenuous, but singleton arrays really do have a special place in Fantasy Land thanks to `of`. This seems natural:

```javascript
> Z.duplicate(Z.of(Identity, 42))
Identity(Identity(42))

> Z.duplicate(Z.of(Array, 42))
[[42]]
```

I've changed my mind. I now prefer your order. :)

Reply:
Let's change the second example to this:

```javascript
> duplicate([1])
[[1]]
```

This will create a natural progression from Identity value to singleton array to array with multiple elements.

Reply:
Done